Source/WebCore/platform/mediastream/mac/ScreenCaptureKitSharingSessionManager.h

```diff
@@ -94,7 +94,7 @@ class ScreenCaptureSessionSource
     CleanupFunction m_cleanupFunction;
 };
 
-class ScreenCaptureKitSharingSessionManager : public CanMakeWeakPtr<ScreenCaptureKitSharingSessionManager> {
+class ScreenCaptureKitSharingSessionManager : public RefCountedAndCanMakeWeakPtr<ScreenCaptureKitSharingSessionManager> {
```

Comment:
If we make it RefCounted, I would tend to make ScreenCaptureKitSharingSessionManager constructor private.
And ScreenCaptureKitSharingSessionManager::singleton would probably use a NeverDestroyed Ref.